src/HomePage/NavBar/NavBar.js

```diff
@@ -5,18 +5,21 @@ import { Link } from "react-router-dom";
 const NavBar = () => {
   return (
     <nav className="navbar">
-      <div className="Logo" />
-      <ul>
-        <li>
-          <Link to="/">Home</Link>
-        </li>
-        <li>
-          <Link to="/portfolio">Portfolio</Link>
-        </li>
-        <li>
-          <Link to="/contact">Contact Me</Link>
-        </li>
-      </ul>
+      <Link to="/" className="navbar__link">
+        Home
+      </Link>
+      <Link to="/portfolio" className="navbar__link">
+        Portfolio
+      </Link>
+      <div>
+        <a href="/" className="Logo" />
```

Comment:
This isn't part of this commit but im intrigued as to why your logo has a display of flex when it has no contents inside of it, i just noticed it and thought it was weird (but wont push on it as its not part of this pr)

Reply:
Good point, must of left it in from experimenting before. However, just tried to remove and now it no longer appears... will have to look into this a little more